jupyter_resource_usage/static/main.js

```diff
@@ -56,6 +69,28 @@ define([
                 $('#jupyter-resource-usage-mem').text(display);
             }
         });
+        $.getJSON({
+            url: utils.get_body_data('baseUrl') + 'api/metrics/v1',
+            success: function (data) {
+                var cpuPercent = data['cpu_percent'];
+                var maxCpu = data['cpu_count'];
+                var limits = data['limits'];
+                var display = parseFloat(cpuPercent).toFixed(0) + "% (";
+
+                if (limits['cpu']) {
+                    if (limits['cpu']['cpu']) {
+                        display += (Math.round(parseFloat(cpuPercent) / 100)).toString() + " / " + maxCpu + ")";
+                    }
+                    if (limits['cpu']['warn']) {
+                        $('#jupyter-resource-usage-display-cpu').addClass('jupyter-resource-usage-warn');
+                    } else {
+                        $('#jupyter-resource-usage-display-cpu').removeClass('jupyter-resource-usage-warn');
+                    }
+                }
+
+                $('#jupyter-resource-usage-cpu').text(display);
```

Comment:
I think CPU usage still needs to be explicitly enabled for now with `track_cpu_percent`.

What happens if it is not enabled? Is the "CPU:" label displayed on the UI?

Reply:
Thanks, yeah it wasn't handling that and the display would be messed up. I've committed a change to handle this.